ui/theme.ts

```diff
@@ -43,8 +44,8 @@ const theme = {
       900: '#090e10',
     },
     black: '#000000',
-    green: isCareEnv ? '#F5B743' : '#28EE5F',
-    pink: isCareEnv ? '#F5B743' : '#EA28EE',
+    green: isRcvrEnv ? '#28EE5F' : highlightColor,
+    pink: isRcvrEnv ? '#EA28EE' : highlightColor,
```

Comment:
Maybe we can have `primaryHighlight` and `secondaryHighlight` so we can still put this into `whitelabel.ts`?

Reply:
Sure. I'll order some :)